tests/cypress/integration/html/generateHTML.spec.ts

```diff
@@ -26,4 +27,15 @@ describe('generateHTML', () => {
 
     expect(html).to.eq('<p>Example Text</p>')
   })
+
+  it('can convert from & to html', async () => {
+    const extensions = [Document, Text, Youtube]
+    const html = `<p>Tiptap now supports YouTube embeds! Awesome!</p>
+      <div data-youtube-video>
+        <iframe src="https://www.youtube.com/watch?v=cqHqLQgVCgY"></iframe>
+      </div>`
+    const json = generateJSON(html, extensions)
+
+    expect(generateHTML(json, extensions)).to.equal(html)
+  })
```

Comment:
I wish we had more tests to give more confidence here 😓

Reply:
Could we get an update here? @haydenbleasel (the branch also needs to get synced with upstream again)